### PR TITLE
feat: show labs on Profile page

### PR DESCRIPTION
**Pull request by ana-yldio**

### CODE REVIEW

File: apps/auth0-rules/src/__tests__/add-user-metadata.test.ts [hunk 1]
```diff
@@ -93,6 +93,7 @@ const apiUser: UserMetadataResponse = {
   questions: [],
   role: 'Grantee',
   algoliaApiKey: 'test-api-key',
+  labs: [],
```
Comment (peterstarling) [inline apps/auth0-rules/src/__tests__/add-user-metadata.test.ts:96]: i dont think we want to add labs to auth0 token - i already think we have too much information on it. You can easily omit this field in `UserMetadataResponse` type

File: packages/model/src/user.ts [hunk 2]
```diff
@@ -81,6 +81,11 @@ export interface OrcidWork {
   lastModifiedDate: string;
 }
 
+export interface UserLab {
```
Comment (peterstarling) [inline packages/model/src/user.ts:84]: to be fair i dont think this needs to be a user-specific type.

Labs unlike other entities are not being returned on their own, and so there is no direct Response type definition inside /entities - I would still probably create a file and have a basic Lab definition there that we might use for teams etc.

File: apps/asap-server/test/fixtures/users.fixtures.ts [hunk 3]
```diff
@@ -361,6 +364,7 @@ export const userResponse: UserResponse = {
   questions: ['Question 1', 'Question 2'],
   avatarUrl: `${config.baseUrl}/api/assets/${config.appName}/squidex-asset-id`,
   role: 'Grantee',
+  labs: [],
```
Comment (peterstarling) [inline apps/asap-server/test/fixtures/users.fixtures.ts:367]: thats not really a great fixture if you ask me - you dont need to implement anything for this test to pass, we could just return an empty array. It's ok to test that case too - as in - the empty result, but I would first make sure that we get actual labs here, that goes for both route testing and controller testing (both of which use this fixture)

**Review by peterstarling — CHANGES_REQUESTED** [review]
